SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalancing/ReferenceCountedLockingPolicy/SpatialStaticComponentViewMock.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Interop/SpatialStaticComponentView.h"
+#include "SpatialCommonTypes.h"
+#include "WorkerSDK/improbable/c_worker.h"
+
+#include "SpatialStaticComponentViewMock.generated.h"
+
+UCLASS()
+class USpatialStaticComponentViewMock : public USpatialStaticComponentView
+{
+	GENERATED_BODY()
+public:
+	void Init(Worker_EntityId EntityId, Worker_Authority InAuthority, VirtualWorkerId VirtWorkerId);
```

Comment:
Nit - in on some inputs and not others. Personally I don't think `In` needs to prefix anything that isn't a reference.